src/screens/AddMedicineScreen.tsx

```diff
@@ -0,0 +1,152 @@
+import React, {useState, useRef} from 'react'
+import {
+  SafeAreaView,
+  View,
+  StyleSheet,
+  TextInput,
+  TouchableWithoutFeedback,
+  Image,
+} from 'react-native'
+import {KeyboardAwareScrollView} from 'react-native-keyboard-aware-scroll-view'
+import {RouteProp} from '@react-navigation/native'
+import {StackNavigationProp} from '@react-navigation/stack'
+import {useIntl} from 'react-intl'
+
+import {containerStyles, colors, medicinePill} from '../styles'
+import SCREENS from '../constants/screens'
+import {MEDICINES} from '../constants/medicines'
+import {RootStackParamList} from '../Navigation'
+
+import {useThunkDispatch} from '../redux/store'
+import {BodyText} from '../components'
+
+type AddMedicineScreenNavigationProp = StackNavigationProp<
+  RootStackParamList,
+  SCREENS.ADD_MEDICINE
+>
+
+type AddMedicineScreen = RouteProp<RootStackParamList, SCREENS.ADD_MEDICINE>
+
+type Props = {
+  navigation: AddMedicineScreenNavigationProp
+  route: AddMedicineScreen
+}
+
+function AddMedicineScreen({navigation, route}: Props) {
+  const intl = useIntl()
+  const dispatch = useThunkDispatch()
+
+  const [input, setInput] = useState<string>('')
+  const inputRef = useRef<null | any>(null)
+
+  return (
+    <KeyboardAwareScrollView style={{flex: 1, backgroundColor: colors.grey4}}>
+      <SafeAreaView style={[containerStyles.fill]}>
```

Comment:
The SafeAreaView should be the outmost view in the hierarchy. It shouldn't be within the scroll.